src/v2/styles/Modal/Modal-theme.scss

```diff
@@ -24,6 +24,10 @@
     @include utils.component-layer-overrides('modal');
   }
 
+  .str-chat__modal__inner.str-chat-react__modal__inner {
+    background-color: transparent;
```

Comment:
Is this selector also used for the message edit modal? Because I thought we only want the transparent background for the image carousel modal

Reply:
@szuperaz thank you for spotting this. I have fixed it and reduced it only to image gallery modal